packages/common/src/components/logos/Vue.tsx

```diff
@@ -1,13 +1,13 @@
-import React from 'react';
+import * as React from 'react';
```

Comment:
You have the pragma set up si `import React from 'react'`should work no?

Reply:
Yeah as alternative I could've enabled `esModuleInterop` for TS, but I felt like I was on a roll so I changed everything 😅. No big opinion on this.

Reply:
I caved 😂 https://github.com/CompuIves/codesandbox-client/pull/1573/commits/33f64407d8396a1ac3fa7bc7510504634f31f4a6

Reply:
😍